build.gradle

```diff
@@ -125,22 +128,21 @@ bintray {
 
 ext {
     reformLogging = '2.0.2'
-    springCloudVersion = 'Edgware.RELEASE'
+    springCloudVersion = 'Finchley.M9'
 }
 
 dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
-    compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-feign'
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-openfeign'
```

Comment:
As you are updating the Feign client I would like to see a test where it works with `service-auth-provider-app`.i.e  you should be able to retrieve JWT token using `/lease` endpoint.

Reply:
What a pity there isn't one already 😉

Reply:
What exactly would you like to see @nitinprabhu ? Spring integration test against a Wiremock or a functional test in `service-auth-provider` service pipeline?

Reply:
Checking the functional/smoke tests suite in [service-auth-provider](https://github.com/hmcts/service-auth-provider-app/blob/master/api/src) and I wouldn't call it extensive 😉

Reply:
Yes there is nothing when I was updating it if you see I released many version of `service-auth-provider-java-client` and then realised it is not working.Any test(may be even local) which uses this updated client version to access `service-auth-provider-app` and able to retrive token will do.Hope you don't have to release version of client just to test :)

Reply:
tested locally